spotty/spotty/Data/Models/Response/AlbumImagesWrapper.swift

```diff
@@ -10,5 +10,10 @@ import Foundation
 struct AlbumImagesWrapper: Codable, Hashable {
     
     let uuid = UUID() // TODO: Remove!!!
-    let images: [ImageResponse]
+    private let images: [ImageResponse]
+    
+    var imageURL: String {
+        guard let image = images[safeAt: 0] else { return "" }
```

Comment:
Maybe we can refactor to _images[safeAt: 0]?.url ?? ""_

Reply:
done